yt_dlp/extractor/mbn.py

```diff
@@ -0,0 +1,89 @@
+import re
+
+from .common import InfoExtractor
+from ..utils import (
+    int_or_none,
+    traverse_obj,
```

Comment:
Same as the other review, probably better to use `from ..utils.traversal import traverse_obj`